.golangci.yaml

```diff
@@ -37,11 +37,6 @@ linters-settings:
     min-len: 3
     # minimal occurrences count to trigger, 3 by default
     min-occurrences: 3
-  depguard:
```

Comment:
We need to disable this as Snyk provides much better dependency scanning AND depguard introduced breaking changes. https://github.com/golangci/golangci-lint-action/issues/442#issuecomment-1099890561 re: changes.